pkg/ipmi/blobs/blob_transfer.go

```diff
@@ -319,7 +319,7 @@ func (h *BlobHandler) BlobRead(sid SessionID, offset, size uint32) ([]uint8, err
 // |sessionID| returned from BlobOpen gives us the open blob.
 // |data| is bounded by max size of an IPMI packet, which is platform-dependent.
 // If not all of the bytes can be written, this operation will fail.
-func (h *BlobHandler) BlobWrite(sid SessionID, offset int32, data []int8) error {
+func (h *BlobHandler) BlobWrite(sid SessionID, offset uint32, data []uint8) error {
```

Comment:
data should be a `[]byte`.

Reply:
> data should be a `[]byte`.
byte is an alias for uint8. Should stick with uint8, as uint8 is used everywhere in [pkg/ipmi](https://github.com/u-root/u-root/tree/master/pkg/ipmi)?

Reply:
I think I agree with PlotCondor here? ipmi package is all uint8